tests/components/test_proximity.py

```diff
@@ -7,21 +7,89 @@
 
 import homeassistant.core as ha
 from homeassistant.components import proximity
+from homeassistant.components import zone
+import homeassistant.components.device_tracker as device_tracker
+import homeassistant.util.dt as dt_util
+from datetime import datetime, timedelta
 
 class TestProximity:
     """ Test the Proximity component. """
 
     def setup_method(self, method):
         self.hass = ha.HomeAssistant()
```

Comment:
You will also have to clean up the known devices file after the test is done using a tear down method